app/controllers/api/v1/search_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchController < ApplicationController
+      DEPRECATION_MESSAGE = 'This API endpoint has been deprecated. Please refer to https://search.gov/developer/ for information on current Search.gov APIs.'
+
+      def search
+        render plain: DEPRECATION_MESSAGE, status: :not_found
+      end
+    end
+  end
+end
```

Comment:
This file is actually just `app/controllers/api_controller.rb`, which I've moved (to get it out of the top-level controllers directory), renamed, and cleaned up.